src/smtsolvers/CoreSMTSolver.cc

```diff
@@ -2363,6 +2348,18 @@ void CoreSMTSolver::clearSearch()
 //    }
 }
 
+lbool CoreSMTSolver::zeroLevelConflictHandler() {
```

Comment:
I like this change, but I hope it'll be temporary.  I'd like to have the scatter-based splitting in a derived class.

Reply:
That would be great.